azext_edge/tests/edge/orchestration/resources/connector/opcua/test_opcua_certs_client_unit.py

```diff
@@ -58,6 +59,49 @@
                 resource_group_name="mock-rg",
                 objects="new-secret",
             ),
+            {
+                "configurationSettings": {
+                    "connectors.values.securityPki.applicationCert": OPCUA_CLIENT_CERT_SECRET_SYNC_NAME,
```

Comment:
We need a test where other config settings exist, to assert only connectors settings are in play for the PATCH. We don't want to [re-]apply any other setting. That should help inform the implementation.

Reply:
thanks! test update